gammapy/image/profile.py

```diff
@@ -194,6 +196,241 @@ def save(self, filename):
         raise NotImplementedError
 
 
+class ImageProfile(object):
+    """
+    Image profile class.
+
+    The image profile data is stored in `~astropy.table.Table` object, with the
```

Comment:
Does this work for tables with columns that don't have units?
Or are there certain requirements on units for certain columns?

(my suggestion would be to **not** hard-code ImageProfile specific behaviour, and make this more of a general Profile1D class)